typegraph/core/src/typegraph.rs

```diff
@@ -110,6 +110,9 @@ pub fn init(params: TypegraphInitParams) -> Result<()> {
             .collect::<Vec<_>>()
     };
 
+    // the default runtime is the deno runtime that will be registered below
+    let default_runtime_idx = 0;
```

Comment:
I would not take any assumption on the id the deno runtime, in the future it might be something else?

Reply:
Ok